auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -339,6 +342,11 @@ export default defineComponent({
       getHeaders(newCol)
     })
 
+    watch(() => affiliations.results, () => {
```

Comment:
I think this will fix an issue related to filters. If we added a new row when some column was hidden, the table updated with all the columns shown, which was not consistent with the `Columns to Show` checkbox.

![image](https://github.com/bcgov/sbc-auth/assets/60866283/b9b824bd-f272-43a8-afef-995be036e7a8)